setup.py

```diff
@@ -49,8 +49,26 @@
     ]},
     install_requires=['pytz'],
     extras_require={
-        "bcrypt": ["bcrypt"],
-        "argon2": ["argon2-cffi >= 16.1.0"],
+        'admindocs': ['docutils'],
+        'argon2': ['argon2_cffi>=16.1.0'],
+        'bcrypt': ['bcrypt'],
+        'gis': [
+            'geoip2',
+            'numpy',
+        ],
+        'geoip': ['geoip2'],
+        'jinja2': ['Jinja2>=2.9.2'],
+        'memcached': ['python3-memcached'],
```

Comment:
Just wanted to point out https://github.com/django/django/commit/770b9ea77fb5e39d616e62b54c06755e6d4f4d36 which was committed recently.